packages/polymer-legacy-adapter/src/template-renderer-templatizer.js

```diff
@@ -142,4 +143,4 @@ export class Templatizer extends PolymerElement {
   }
 }
 
-customElements.define(Templatizer.is, Templatizer);
+defineCustomElement(Templatizer);
```

Comment:
This class is missing `static get version()` - we might want to add one for consistency.